concrete/src/Api/OAuth/Controller.php

```diff
@@ -128,9 +129,7 @@ public function authorize()
                 // Otherwise if consent is disabled just mark this request as approved.
                 $request->setAuthorizationApproved(true);
                 $this->storeRequest($request);
-
-                // Update the step
-                $step = $this->determineStep($request);
+                return $this->oauthServer->completeAuthorizationRequest($request, $response);
```

Comment:
All this looks to be doing is removing the call to `$this->clearRequest($request)`, why is that? If we don't clear the request from session it's possible for someone to resume the authorization flow in unexpected ways